tcms/testruns/templates/testruns/get.html

```diff
@@ -0,0 +1,102 @@
+
+{% block contents %}
+
+<div class="container-cards-pf">
+    <!-- Important:  if you need to nest additional .row within a .row.row-cards-pf, do *not* use .row-cards-pf on the nested .row  -->
+    <h1 class="col-md-12" style="margin-top: 0">
+        <span id="test_case_pk"
+            data-pk="{{ object.pk }}"
+            data-perm-remove-tag="{{ perms.testcases.delete_testcasetag }}"
+            data-perm-remove-component="{{ perms.testcases.delete_testcasecomponent }}"
+            data-perm-remove-plan="{{ perms.testcases.delete_testcaseplan }}"
+            data-perm-remove-bug="{{ perms.testcases.delete_bug }}"
+        >TR-{{ test_run.pk }}:</span> {{ test_run.summary }}
+    </h1>
+</div>
+
+<div class="col-xs-12 col-sm-12 col-md-3">
+    <div class="card-pf card-pf-accented card-pf-aggregate-status">
+
+        <h2 class="card-pf-title" style="text-align: left">
+            <span class="fa fa pficon-topology"></span>{% trans 'Test plan' %}:
+            <a href="{% url 'test_plan_url_short' test_run.plan.pk %}">[{{ test_run.plan.pk }}] {{ test_run.plan.name }}</a>
+        </h2>
+
+        <h2 class="card-pf-title" style="text-align: left">
+            <span class="fa fa-shopping-cart"></span>{% trans 'Product' %}:
+            <a href="{% url 'testruns-search' %}?product={{ test_run.product_version.product_id }}" title="Search test runs of {{ test_run.product_version.product }}">{{ test_run.product_version.product }}</a>
+        </h2>
+
+        <h2 class="card-pf-title" style="text-align: left">
+            <span class="fa fa-search"></span>{% trans 'Product version' %}:
+            <a href="{% url 'testruns-search' %}?product={{ test_run.plan.product_id }}&product_version={{ test_run.plan.product_version_id }}" title="Search test runs of {{ test_run.plan.product_version.value }}">{{ test_run.plan.product_version.value }}</a>
+        </h2>
+
+        <h2 class="card-pf-title" style="text-align: left">
+            <span class="fa fa-cog"></span>{% trans 'Build' %}:
+            <a href="{% url 'testruns-search' %}?product={{ test_run.plan.product_id }}&product_version={{ test_run.plan.product_version_id }}&build={{ test_run.build_id }}" title="Search test runs of {{ test_run.build_id }}">{{ test_run.build }}</a>
+        </h2>
+
+        <h2 class="card-pf-title" style="text-align: left">
+            <span class="fa pficon-user"></span>{% trans 'Manager' %}:
+            <a href="{% url 'tcms-profile' test_run.manager.username %}">{{ test_run.manager.username }}</a>
+        </h2>
+
+        <h2 class="card-pf-title" style="text-align: left">
+            <span class="fa pficon-user"></span>{% trans 'Default tester' %}:
+            <a href="{% url 'tcms-profile' test_run.default_tester.username %}">{{ test_run.default_tester.username }}</a>
+        </h2>
+
+        {% comment %} TODO: buttons are still not working {% endcomment %}
+        {% comment %} TODO: better styling for buttons {% endcomment %}
```

Comment:
No TODOs, see reasoning above. 

Missing functionality is fine, e.g. release with reduced functionality if we feel like it.

Reply:
I agree with the above reasoning, but don't see why I should remove all the TODOs. These are still thinks that are going to be done later, so what's the harm of having them marked in that fashion?